src/libs/ReportUtils.ts

```diff
@@ -3065,10 +3066,41 @@ function getInvoicePayerName(report: OnyxEntry<Report>): string {
     return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${invoiceReceiver?.policyID}`]);
 }
 
+/**
+ * Parse html of reportAction into text
+ */
+function parseReportActionHtmlToText(reportAction: OnyxEntry<ReportAction>, reportID: string): string {
+    const {html, text} = reportAction?.message?.[0] ?? {};
+
+    if (!html) {
+        return text ?? '';
+    }
+
+    const mentionReportRegex = /<mention-report reportID="(\d+)" *\/>/gi;
+    const matches = html.matchAll(mentionReportRegex);
+
+    const reportIdToName: Record<string, string> = {};
```

Comment:
```suggestion
    const reportIDToName: Record<string, string> = {};
```

Reply:
@luacmartins I am following the ExpensiMark field name in [here](https://github.com/Expensify/expensify-common/blob/c3cc2c0d07c961df08a7147b9c4405014521b8bd/lib/ExpensiMark.js#L539).

If I use reportIDToName it won't work.